terraform/terraform-create-resource/main.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_version = ">=0.12"
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "~>2.91.0"
+    }
+  }
+
+  backend "azurerm" {
+        resource_group_name  = "StateResource"
+        storage_account_name = "couplemgmtstates"
+        container_name       = "tfstates"
+        key                  = "test.terraform.tfstate"
+    }
+}
+
+resource "random_uuid" "test" {
+}
```

Comment:
does the auto-indentation cause this statement to take up 2 lines? Just checking